packages/desktop-client/src/components/budget/util.ts

```diff
@@ -174,17 +174,3 @@ export async function prewarmAllMonths(
     months.map(month => prewarmMonth(budgetType, spreadsheet, month)),
   );
 }
-
-export async function switchBudgetType(
```

Comment:
Not necessary anymore IMO. We already do the prewarming operation in the budget page - inside a `useEffect` hook. So whenever `budgetType` changes - the prewarming will be performed.